source/whats-new.txt

```diff
@@ -57,8 +57,38 @@ The 3.0 driver release includes the following new features:
   This type is available in .NET 5 and later.
   To learn more about the ``Half`` type, see the
   `Half Struct <https://learn.microsoft.com/en-us/dotnet/api/system.half?view=net-8.0>`__
-  API reference page on MSDN. 
-
+  API reference page on MSDN.
+
+- Adds support for implicit client-side projection when using the ``Find()`` and
+  ``Select()`` methods. In previous versions of the driver, you could perform client-side
```

Comment:
Beside the `Find` and `Select` methods this will apply to the `Project` pipeline stage builder (which is analogous to the `Select` method).